assets/js/functionality.js

```diff
@@ -51,10 +105,8 @@ $(window).on('load', function() {
                 }
             }
             if (currentSection) {
-                let id = currentSection.getAttribute('id');
-                document
-                    .querySelector(`nav li a[href="#${id}"]`)
-                    .parentElement.classList.add('active');
+                const overviewItem = findExistingSectionInOverview(currentSection);
+                overviewItem && overviewItem.classList.add('active');
             }
         }
     });

```

Comment:
In common file `assets/js/functionality.js` we don't have access exporter property `tocHideHeading3` because this file is copied as is and is not processed by Pulsar